dataclasses_avroschema/model_generator/lang/python/base.py

```diff
@@ -185,7 +196,8 @@ def render_class(self, *, schema: JsonDict) -> str:
             docstring=docstring,
         )
 
-        class_metadata = self.render_metaclass(schema=schema, field_order=field_order)
+        add_schema_name = not name == schema["name"]
```

Comment:
You can do: `add_schema_name = name != schema["name"]`

Reply:
Agreed!